### ISSUE TITLE
Invite current users of GetTogether to private groups

### ISSUE THREAD

**Issue by craigmaloney** [GetTogetherComm/GetTogether#220]
at the moment the only way to add users to a private group is via email. This has proved less than optimal for one of our private groups as there are still outstanding invitations showing for members that have somehow already joined the group.

Would it be possible to have a way to invite a user (via their profile page) to a private group.

One thing to keep in mind is that allowing anyone to invite any user to a group might be problematic, so perhaps a check for both users to be in a group would help reduce invite spam and harassment.

**Comment by mhall119** [comment]
Even with the suggested limitation I'm concerned about the potential to spam people. It wouldn't be difficult to write a bot to join a team that's listed on the user's profile before sending them the invite to a private spam team

**Comment by craigmaloney** [comment]
Indeed, but right now it's rather difficult to get folks into a private group. I've half-considered making the group public to work around the wonky invite system and then making it private when folks are done joining.

**Comment by mhall119** [comment]
Another option would be to generate an "Invite link" that you could give to anybody to join the team. That would potentially allow non-invited people to join if they get access to the link, but we could give controls to regenerate it or disable it at any time if that happens.

**Comment by craigmaloney** [comment]
That makes sense. Then it's on the person generating the link and those who have the link to abide the wishes of everyone involved. I like it.
